src/mainboard/pcengines/apu2/mainboard.c

```diff
@@ -359,3 +363,13 @@ struct chip_operations mainboard_ops = {
 	.enable_dev = mainboard_enable,
 	.final = mainboard_final
 };
+
+void change_build_date_format()
+{
+	char tmp[15];
+	strcpy(tmp, coreboot_dmi_date);
+	strncpy(coreboot_dmi_date,   tmp+6, 4);
+	strncpy(coreboot_dmi_date+4, tmp+3, 2);
+	strncpy(coreboot_dmi_date+6, tmp,   2);
```

Comment:
@krystian-hebel this looks like magic dance. We should have some comment here what we do and why.

Reply:
Done.